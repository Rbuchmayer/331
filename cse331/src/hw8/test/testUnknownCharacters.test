r
cse
kne
r
rob
CSE
r
CSE
kne
q