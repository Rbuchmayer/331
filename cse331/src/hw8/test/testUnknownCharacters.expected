Menu:
	r to find a route
	b to see a list of all buildings
	q to quit

Enter an option ('m' to see the menu): Abbreviated name of starting building: Abbreviated name of ending building: Unknown building: cse
Unknown building: kne

Enter an option ('m' to see the menu): Abbreviated name of starting building: Abbreviated name of ending building: Unknown building: rob

Enter an option ('m' to see the menu): Abbreviated name of starting building: Abbreviated name of ending building: Unknown building: kne

Enter an option ('m' to see the menu):